elip-0100.mediawiki

```diff
@@ -92,7 +90,7 @@ TBD
 
 PSETs containing Hardware Wallet Extensions are considered backwards compatible with both the mainstream PSET format and the BIP 370 PSBT Version 2 format, on which PSET is based.
 
-Existing implementations, following the policy specified in the initial BIP 174 PSBT specification, ignore all unknown fields, and thus should not be affected by the presence of the new fields belonging to Hardware Wallet Extensions. Since none of the new fields alter produced signatures nor the network format transaction, inability to decode them has no functional consequences.
+Existing implementations, following the policy specified in the initial BIP 174 PSBT specification, ignore all unknown fields, and thus should not be affected by the presence of the new fields belonging to Hardware Wallet Extensions. Since none of the new fields affects produced signatures nor the network format transaction, inability to decode these new fields has no functional consequences.
```

Comment:
nit: `affects` -> `affect`.

also, `decode` -> `process` - decoding failures are fatal in PSBT.

Reply:
thanks